marketplace/adoptium-marketplace-schema/src/main/java/net/adoptium/marketplace/schema/HeapSize.java

```diff
@@ -0,0 +1,9 @@
+package net.adoptium.marketplace.schema;
+
+import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+@Schema(type = SchemaType.STRING, enumeration = {"normal", "large"}, example = "normal")
```

Comment:
Not 100% sure on this but I think Large heap size concept has gone away permanently

Reply:
If we are going to maintain backward compatibility for old builds, will need it. If not can get rid of it

Reply:
I think you can drop it